cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java

```diff
@@ -60,6 +60,8 @@ public final class DataprocUtils {
   public static final Path CACHE_DIR_PATH = Paths.get(System.getProperty("java.io.tmpdir"),
                                                       "dataproc.launcher.cache");
   public static final String LOCAL_CACHE_DISABLED = "disableLocalCaching";
+  public static final String DATAPROC_TROUBLESHOOTING_DOCS_URL =
+    "https://cloud.google.com/dataproc/docs/troubleshooting";
```

Comment:
Did you check if this doc covers the error codes you are catching here? If not please file a P1 doc bug on Dataproc and add it to the hotlist b/4464054?

Reply:
It's better to make this link configurable via the dataproc provisioner system property (settable via cdap-site.xml)

Reply:
Enabled setting the docs link from cdap-site. 
Ack regarding bug for updates to dataproc docs.